installing_cloudforms_on_red_hat_enterprise_linux_openstack_platform/installing_cloudforms.adoc

```diff
@@ -0,0 +1,135 @@
+[[installing-cloudforms]]
+== Installing {product-title}
+
+{product-title} is able to be installed and ready to configure in a few quick steps. After downloading {product-title} as a virtual machine image template from the {product-title} Customer Portal, the installation process takes you through the steps of uploading the appliance to a supported virtualization or cloud provider.
+
+[IMPORTANT]
+=======
+After installing the {product-title} Appliance, you must configure the database for {product-title}. See xref:configuring_a_database[].
+=======
+
+=== Obtaining the {product-title} Appliance
+
+. Go to link:https://access.redhat.com[access.redhat.com] and log in to the {product-title} Customer Portal using your customer account details.
```

Comment:
This is the wrong link and wrong procedure. The downstream appliances are available here:
http://manageiq.org/download/